Polyfills/Canvas/Source/Font.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <string>
+#include <optional>
+
+namespace Babylon::Polyfills::Internal
+{
+    enum class FontStyle
+    {
+        Normal,
+        Italic,
+    };
+
+    struct Font
+    {
+    private:
+        static constexpr int NORMAL_WEIGHT = 400;
+        static constexpr int BOLD_WEIGHT = 700;
+
+    public:
+        FontStyle Style = FontStyle::Normal;
+        int Weight = NORMAL_WEIGHT;
+        float Size = 10;
+        std::string Family = "sans-serif";
```

Comment:
Should these be immutable? Seems like they could be passed into a constructor and then exposed through get functions?